src/main/java/eu/enexa/docker/ContainerManagerImpl.java

```diff
@@ -25,14 +22,17 @@
 @Component("dockerContainerManager")
 public class ContainerManagerImpl implements ContainerManager {
     private static final Logger LOGGER = LoggerFactory.getLogger(ContainerManagerImpl.class);
-    private static final String VOLUME_PATH = System.getenv("ENEXA_SHARED_DIRECTORY");
-    private static final String HOST_BASE_PATH = System.getenv("ENEXA_SHARED_DIRECTORY");
-
-    //private static final String NETWORK_NAME = "enexaNet";
+    private String hostBasePath ;
+    private String hostWritablePath ;
+    private String hostModuleInstancePath ;
```

Comment:
Only the base path should be an attribute of this class. The other two (writable path and module path), can be different depending in which context the container manager is used. Since we have only one manager per service, it has to be able to handle requests coming from all different applications without changing its internal state (since that would make it not thread-safe).